styles.css

```diff
@@ -7,51 +7,46 @@ body {
   height: 100%;
   margin: 0;
   background-color: #e5e5e5;

```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. your spacing(margin, padding) are supposed to be similar to the design template in Figma. as you may notice how your `portal section` overlap in the navbar. 
![image](https://user-images.githubusercontent.com/97100091/227708898-8254945c-e3b7-4fd8-8a74-8157049e728d.png)
`Hint: You might need to give your element a margin-top.`

Reply:
I'm going to give it a top margin but you should take a look at the design first. It is overlapping:

![image](https://user-images.githubusercontent.com/32914511/227713273-06e6f883-1d3e-4f20-8c76-4ed7626272b0.png)